docs/spark-standalone.md

```diff
@@ -328,6 +328,17 @@ SPARK_WORKER_OPTS supports the following system properties:
   </td>
   <td>3.0.0</td>
 </tr>
+<tr>
+  <td><code>spark.shuffle.service.db.backend</code></td>
+  <td>LEVELDB</td>
+  <td>
+    When <code>spark.shuffle.service.db.enabled</code> is true, user can use this to specify the kind of disk-based 
+    store used in shuffle state store. This supports `LEVELDB` and `ROCKSDB` now and `LEVELDB` as default value. 
+    This only affects standalone mode (yarn always has this behavior enabled). 
```

Comment:
Do you mean yarn always enable db or always use LEVELDB as its db? Could you make it more clear?

Reply:
will delete this

Reply:
deleted